llama.cpp

```diff
@@ -955,10 +955,10 @@ struct llama_vocab {
     id linefeed_id = 13;
 
     int find_bpe_rank(std::string token_left, std::string token_right) const {
-        replace_all(token_left,  " ",  "Ġ");
-        replace_all(token_left,  "\n", "Ċ");
-        replace_all(token_right, " ",  "Ġ");
-        replace_all(token_right, "\n", "Ċ");
+        replace_all(token_left,  " ",  "\u0120");
```

Comment:
I like this change, because it makes it clearer that we are just adding 0x100 to these codepoints.

Reply:
Doesn't this remove some of the cases? Am I missing something or should it be:

```c++

        replace_all(token_left,  "\n", "\u010a");
        replace_all(token_right, " ",  "\u0120");
        replace_all(token_right, "\n", "\u010a");
```

Reply:
`\u0120` is `Ġ`
`\u010A` is `Ċ`

It should be the exact same case covered, only with the Unicode escape sequence instead. That's all I changed.

Reply:
Apologies. In the review it just showed:

```patch
-        replace_all(token_left,  "\n", "Ċ");
-        replace_all(token_right, " ",  "Ġ");
-        replace_all(token_right, "\n", "Ċ");
+        replace_all(token_left,  " ",  "\u0120");
```

in the changes so it looked like it was remove 3 cases and only adding one back. Sorry for the confusion.